reactor-netty5-http/src/main/java/reactor/netty5/http/client/HttpClientFinalizer.java

```diff
@@ -31,7 +33,7 @@
 import reactor.netty5.Connection;
 import reactor.netty5.NettyOutbound;
 import reactor.netty5.channel.ChannelOperations;
-
+import reactor.util.annotation.Nullable;
```

Comment:
Should we keep empty row between imports and static imports?

Reply:
added missing empty line, in order to separate imports and static ones.